r/extra-tests/write-parquet.R

```diff
@@ -0,0 +1,23 @@
+library(arrow)
+
+if (!dir.exists("extra-tests/files")) {
+  dir.create("extra-tests/files")
+}
+
```

Comment:
could this source tests/testthat/helper-data.R instead of copying the object here? just so these don't get out of sync (if we add more attrs or columns)